minho/programmers/Level_1/6일차/모의고사.js

```diff
@@ -0,0 +1,24 @@
+function solution(answers) {
+    let supo = [[1, 2, 3, 4, 5], [2, 1, 2, 3, 2, 4, 2, 5], [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]]; 
```

Comment:
```suggestion
    const loserOfMath = [[1, 2, 3, 4, 5], [2, 1, 2, 3, 2, 4, 2, 5], [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]]; 
```
변수명은 길어도 확실한 변수명이 좋고, 배열은 const로 써주는것이 좋아요!